src/core/main/Reflection/TypeNamespaceExtensions.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+
+namespace RapidCore.Reflection
+{
+    public static class TypeNamespaceExtensions
+    {
+        /// <summary>
+        /// Get the root namespace from a Type - i.e. the first part.
```

Comment:
**suggestion**: Change this line, it confused me a bit because I understood as: "This method will return the root namespace".